.github/actions/devcontainer_run_command/action.yml

```diff
@@ -47,55 +47,55 @@ inputs:
     required: true
```

Comment:
Just out of interest - why turn these all to optional?

Reply:
Because we don't send them on every call to this action. 
When I added the Github Actions linter it started to complain that a required input wasn't being passed...

Reply:
Makes sense!